.gitignore

```diff
@@ -0,0 +1,12 @@
+**/.DS_Store
+**/.idea
+
+**/*.iml
+**/*.log
+**/package-lock.json
```

Comment:
Isn't `package-lock.json` meant to be checked in into VCS? Or is it replaced by `yarn.lock`? If it's the latter case, `yarn.lock` is missing in the PR.

Reply:
Since we're installing dependencies from a local npm registry on `workbench-screens` package, I removed package-lock.json/yarn.lock from the installation. I'm bringing it back for the `core` module.